broker/cmd/dejaqcli-consumer/main.go

```diff
@@ -95,7 +95,8 @@ func (c *Config) durationTimeout() time.Duration {
 }
 
 func main() {
-	go exporter.SetupStandardMetricsExporter(subsystemConsumer)
+	go exporter.GetMetricsExporter(subsystemConsumer, "2111")
```

Comment:
why do we need 2 open ports for metrics?

Reply:
we have two registry the export metrics, first are dejaq metrics second are go_service metrics

Reply:
let's merge them please I didn't saw this pattern and is not ok to have 2 ports open only for this. We can push our metrics in the defaults go client space and this way we expose all of them right? We also can have a prefix of dejaq to separate the metrics.